Mathlib/MeasureTheory/Integral/CircleIntegral.lean

```diff
@@ -210,6 +212,19 @@ protected theorem finsum {ι : Type*} {f : ι → ℂ → E} (h : ∀ i, CircleI
 nonrec theorem neg (hf : CircleIntegrable f c R) : CircleIntegrable (-f) c R :=
   hf.neg
 
+/--
+If `f` is circle integrable, then so are its scalar multiples.
+-/
+theorem const_smul {a : A} {f : ℂ → A} (h : CircleIntegrable f c R) :
+    CircleIntegrable (a • f) c R := by
```

Comment:
Can you remove the `by apply` (perhaps adding some _ or `..`)?

Reply:
Also, you could write
```
lemma const_smul {a : A} {f : ℂ → A} (h : CircleIntegrable f c R) : CircleIntegrable (a • f) c R :=
```
with exactly 100 characters. (Or move `{a : A}` into the `variable` above.)

Reply:
Done.